\chapter{Introduction}

Bioenergetics is the study of biological energy flow. All living things are powered by their environment, and it is the conversion of energy from one state to another that defines the organic life-form. All organisms can be categorized into one of two types: autotrophs and heterotrophs. Autotrophs have the ability to produce their own food, such as carbohydrates and other complex molecules, by harnessing the energy of the sun via photosynthesis. Heterotrophs are unable to produce food and must consume other organisms and break down complex molecules in a process called catabolism. Catabolism, also known as cellular respiration, is fundamental to all aerobic life and is made up of some of the most well understood biochemical pathways. It consists of consecutive metabolic reactions that transfer the energy stored in carbohydrates, fats, and protein to ATP (the cell's currency for energy) and to other electron carriers (such as NADH and FADH$_{2}$). Glycolysis, the first step of catabolism in the cell, begins with the breakdown of glucose. This reaction produces the raw materials required for the citric acid cycle (also known as the Krebs or TCA cycle) which in turn produces electron donors for the cell's main source of energy, the electron transport chain (ETC) (see Figure \ref{fig:catabolism}).

When we breathe, oxygen diffuses into the bloodstream, ultimately heading toward the ETC in the mitochondrial inner membranes of our cells to be harvested for its energy. A similar process occurs in the cytoplasmic membrane of bacteria. The primary role of the ETC is to conserve energy in the form of a transmembrane electrochemical potential from the ``negative'' N-side to the ``positive'' P-side of the membrane. In the ETC, a series of redox reactions between electron donors (from the Krebs cycle) and electron acceptors (such as oxygen) generate the free energy required to produce a proton gradient across the membrane \cite{Papa:1976p9824}. This pH gradient gives rise to the ``proton-motive force'' ($\Delta \mu$\ce{H^+}) which is subsequently utilized by ATP synthase for the phosphorylation of ADP in a highly efficient energy-conversion mechanism known as oxidative phosphorylation. This process, whereby the ETC and ATP synthesis are coupled by $\Delta \mu$\ce{H^+}, is known as the chemiosmotic theory. It was proposed in the 1960s by Peter Mitchell \cite{Mitchell:1966p10492}, who received the Nobel Prize in 1978 for its discovery.

\begin{figure}[htbp]
\centering
\includegraphics{figures/introduction/catabolism.png}
\caption[A highly simplified overview of catabolism in eukaryotic cells.]{A highly simplified overview of catabolism in eukaryotic cells. The products from glycolysis feed the Krebs cycle (citric acid cycle) which provides reactants to the electron transport chain (ETC). The ETC occurs in the inner-membrane of the mitochondria and produces the proton gradient that powers ATP synthesis.}
\label{fig:catabolism}
\end{figure}

The proton motive force is generated by the translocation of protons across a membrane. Proton pumps are proteins that perform this function and therefore form the ``links'' of the electron transport chain. The chain consists of a series of four membrane-bound carriers named Complex I through IV. Complexes I, III, and IV facilitate proton translocation across the inner membrane, but complex II does not function as a proton pump at all. Complex I (NADH dehydrogenase) and Complex II (succinate dehydrogenase) catalyze electron transfer from NADH and succinate, respectively, to ubiquinone. Complex III (cytochrome \emph{bc}$_1$ complex) transfers electrons from ubiquinol to cytochrome \emph{c}. Complex IV (cytochrome \emph{c} oxidase), the terminal enzyme in the chain, transfers electrons from reduced cytochrome \emph{c} to \ce{O2}. This final electron transport step serves the important function of removing electrons from the system so that the electron carriers can be re-oxidized.

The overall equation of the electron transport chain can be summarized as:
\begin{center}
\ce{NADH + 11H^+_n + 1/2O2 ->  NAD^+ + 10H^+_p + H2O},
\end{center}
where \ce{H^+_n} are protons on the N-side of the membrane, and \ce{H^+_p} are protons on the P-side. For every pair of electrons transferred to \ce{O2} through the ETC, the P-side gains 10 protons.

This efficient process is what gives our cells the energy to function, but the molecular mechanisms that drive it are not yet well understood. One of the key hurdles in the field of bioenergetics is understanding the function of redox-driven proton pumps at the molecular level. Specifically of current interest is the terminal enzyme in the ETC, cytochrome \emph{c} oxidase. The function of this protein is central to cellular respiration since it contributes significantly to both the proton and charge gradients. It produces the final product of respiration (\ce{H2O}), and in doing so must prevent the creation of reactive oxygen species.

\section{Cytochrome \emph{c} oxidase}

Cytochrome \emph{c} oxidase (C\emph{c}O) is found in both eukaryotes and bacteria. C\emph{c}O is a member of the heme-copper oxidase (HCO) superfamily which is defined by a protein subunit (subunit I) containing a binuclear centre (BNC) composed of a high-spin, oxygen-binding heme and \ce{Cu_B}. A low-spin heme coordinated by two histidines is also found in the same subunit. A \ce{Cu_A} centre, involved in electron transfer from cytochrome \emph{c}, is found in subunit II. C\emph{c}O is also known as an $aa_3$-type oxidase, as it contains a pair of hemes ($a$ and $a_3$). Mammalian C\emph{c}O has 13 subunits \cite{Capaldi:1990p10284,Shimokata:2007p10305}. Bacterial C\emph{c}O is composed of only 4 subunits, although it still functions by a similar mechanism \cite{Hosler:1992p6662,Pfitzner:1998p10266}. Comparison between the mitochondrial and bacterial complexes suggests that the first three subunits are necessary for normal oxidase function \cite{Capaldi:1990p10284}. Compared to their eukaryotic counterparts, bacterial heme-copper oxidases are attractive to study for the following reasons: they are not as structurally complex, they are easier to work with experimentally and they are highly similar in terms of both structure and function \cite{Pfitzner:1998p10266}. X-ray crystal structures of C\emph{c}O from eukaryotic and prokaryotic organisms have been solved \cite{Yoshikawa:1988p6660,Iwata:1995p10222,Tsukihara:1995p8115,Tsukihara:1996p10224,SvenssonEk:2002p5595,Qin:2006p7708,ShinzawaItoh:2007p5268,Muramoto:2007p8383,Qin:2008p7672,Durr:2008p6162,Qin:2009p5745,Aoyama:2009p10495,Koepke:2009p10494,Muramoto:2010p10493,Liu:2011p9691}. Cytochrome \emph{c} oxidase from \emph{Rhodobacter sphaeroides}, a purple bacteria, has been widely used as a bacterial C\emph{c}O model system for almost two decades \cite{Hosler:1992p6662}. The work done in this thesis is based on the structure of \emph{R. sphaeroides} oxidase.

Among the respiratory enzymes, only cytochrome \emph{c} oxidase qualifies as a ``true proton pump'' \cite{Brzezinski:2008p4584}. This was controversially discovered by Mårten Wikström in 1977 \cite{Wikstrom:1977p5515} and it came as a great surprise to researchers (especially to Peter Mitchell), as the enzyme was thought to translocate electrons alone \cite{Moyle:1978p10104}. Protons pumped by a ``true pump'' are distinguished by not being involved directly in the redox chemistry \cite{Capaldi:1990p10284}. In layman's terms, a pump is \emph{an apparatus that moves a fluid in a single direction from one container to another, while overcoming conditions such that the fluid would not move without intervention}. Regardless of how it is designed, a pump is required (a) to have input and output channels, (b) a repeating mechanism, and (c) a valve to prevent back-flow and leakage. In the case of a molecular pump that moves protons against an electrochemical gradient ($\Delta \mu$\ce{H^+}), its mechanism should maintain the directionality of proton flow while preventing back-flow \cite{Brzezinski:2008p4584}. C\emph{c}O functions with such a mechanism, the details of which are still controversial \cite{Qin:2009p5745}.

The enzyme repeatedly catalyzes an exergonic (energy releasing) redox reaction in order to pump protons across a membrane against an electrochemical gradient \cite{Brzezinski:2003p9848}. The overall reaction can be summarized as:
\begin{center}
\ce{O2 + 4e^-_p + 8H^+_n -> 2H2O + 4H^+_p}
\end{center}
where the subscripts $n$ and $p$ indicate the N-side and P-side, respectively. During each run of the enzyme's catalytic cycle, also called a ``turnover'', four electrons are donated from cytochrome \emph{c} on the P-side and eight protons are taken-up from the N-side, half of which are pumped across the membrane. The remaining protons are consumed during the reduction of oxygen to produce two water molecules.

% we have now specified the requirements of the pump, now we can describe what we know of the catalytic cycle
\subsection{The catalytic cycle}

In cytochrome \emph{c} oxidase, oxygen is reduced to water in a series of steps that define the catalytic cycle. This cycle has been thoroughly studied since the late 1980s, when the first models coupling electron and proton movement throughout the redox cycle were published \cite{Wikstrom:1988p10232}. As more experimental data has become available over the past two decades, many improved models have been proposed \cite{Michel:1999p5651,Wikstrom:2000p9864,Wikstrom:2002p5573,Nyquist:2003p5405,Bloch:2004p5454,Gennis:2004p10239,Verkhovsky:2006p5339,Fadda:2008p5482}. The redox chemistry takes place in the binuclear centre (BNC), home to heme $a_3$ and \ce{Cu_B}, and in close proximity to heme $a$. All models divide the cycle into a series of distinct states indicating (by convention) the redox state of the BNC (heme $a_3$ and \ce{Cu_B}), as seen in Figure \ref{fig:catalytic_cycle}.
\begin{figure}[htbp]
\centering
\includegraphics{figures/introduction/catalytic_cycle.png}
\caption[The catalytic cycle of cytochrome \emph{c} oxidase.]{The catalytic cycle of C\emph{c}O as proposed by Fadda et al. (2008) \cite{Fadda:2008p5482}. The cycle shows the ``conventional'' redox states (defined in the box on the right) as well as intermediate states in which protons and electrons are consumed (inner arrows). Proton ejection states are also shown (straight arrows). The green arrow indicates the direction of the cycle. Protons subscripts indicate the channel that provides them to the active site. Each state is labeled by the total charge of heme $a$ and the BNC (left and right, respectively): [0$\mid$1], [1$\mid$0], and [1$\mid$1].}
\label{fig:catalytic_cycle}
\end{figure}

The work in this thesis is based on the model by Fadda et al. (2008) \cite{Fadda:2008p5482}, which defines a sequence of electron and proton transfer steps and rationalizes the pumping mechanism in terms of electrostatic coupling of proton translocation and redox chemistry. The model by Fadda et al. (2008) \cite{Fadda:2008p5482} defines the catalytic cycle as the periodic repetition of a sequence of three states differing in the spatial distribution of charge in the active site (Figure \ref{fig:catalytic_subcycle}). Like clockwork, the alternating sequence of redox states enables the timely transfer of electrons from the P-side and protons from the N-side. Each state is labeled by the total charge of heme $a$ and the BNC (left and right, respectively): [0$\mid$1], [1$\mid$0], and [1$\mid$1]. This sequence recurs exactly four times per turnover despite differences in the redox chemistry (see Figure \ref{fig:catalytic_subcycle}).

\begin{figure}[htbp]
\centering
\includegraphics{figures/introduction/subcycle.png}
\caption[The three-state sequence recurring four times per turnover in the catalytic cycle of cytochrome \emph{c} oxidase.]{The three-state sequence recurring four times per turnover in the catalytic cycle of cytochrome \emph{c} oxidase. The three states, defined by the charge distribution of the two active site moieties (heme $a$ and the BNC), trigger: proton/electron uptake, proton/electron transfer, and proton ejection. This figure depicts the $O \rightarrow O_M \rightarrow O_R$ state transitions as defined by Fadda et al. (2008) \cite{Fadda:2008p5482}.}
\label{fig:catalytic_subcycle}
\end{figure}

The three-state sequence begins with the transition between state [1$\mid$1] to state [0$\mid$1], as the reduction of heme $a$ (by cytochrome \emph{c}) is accompanied by the uptake of a proton. Next, the BNC (heme either $a_3$ or Cu$_B$) is reduced via electron transfer from heme $a$, resulting in state [1$\mid$0]. This prompts the transfer of a proton to the BNC, producing state [1$\mid$1]. The resultant maximization of positive charge within the enzyme, causes the ejection of a proton from the proton loading site (PLS) \cite{Fadda:2008p5482}. The cycle repeats as heme $a$ is reduced again.

% end of the catalytic cycle
% the requirements of the catalytic cycle, now describe the proton pathways and the PLS
The cyclic nature of the catalytic mechanism relies on protons and electrons being in the right places at the right times. Whereas electrons can tunnel rapidly between nearby redox centers, protons can diffuse through channels along a ``water wire'', a chain of water molecules, via a Grotthuss-like mechanism whereby protons ``hop'' from hydrogen bond donors to hydrogen bond acceptors \cite{Pomes:1996p5488}. C\emph{c}O must therefore have well-defined proton pathways. These pathways must be able to supply the active site (BNC) with chemical protons and electrons, as well as the proton loading site (PLS) for vectorial protons. Electron-coupled proton transfer reactions provide a strong driving force to pull protons into the BNC, so vectorial protons must somehow avoid being consumed by the chemistry \cite{Gennis:2004p10239}. The proton affinity of the PLS must therefore be modulated by the catalytic cycle. As an analogy, the PLS is like an ``elevator with two doors.'' When a vectorial proton travels up the D-channel, it enters through one door of the elevator. Later in the catalytic cycle, the charge distribution of the active site changes such that the elevator goes up, the other door opens, and the proton exits (through the exit channel). Neither the location, nor the chemical nature of the ``elevator'', nor the mechanism controlling kinetic gating (the ``doors'') are known. However, it has been proposed that the region surrounding the heme propionate groups contains the PLS \cite{Behr:2000p5260,Qian:2004p10319,Mills:2005p5442}.

\begin{figure}[htbp]
\centering
\includegraphics{figures/introduction/schematic.png}
\caption[Schematic depiction of bacterial cytochrome \emph{c} oxidase adapted from the X-ray crystal structure of \emph{R. sphaeroides} oxidase.]{Schematic depiction of bacterial cytochrome \emph{c} oxidase adapted from the X-ray crystal structure of \emph{R. sphaeroides} oxidase \cite{SvenssonEk:2002p5595}. The enzyme catalyzes a redox reaction in the binuclear centre (BNC) between dioxygen, electrons, and protons. Dioxygen diffuses into the BNC through the membrane. Electrons are donated from cytochrome \emph{c} (on the P-side) to the Cu$_A$ site. Chemical and vectorial protons enter through the K- and D-channels. Vectorial protons are transferred first to the proton loading site (PLS) from residue E286 prior to exiting via the exit channel. The exact locations of the PLS and exit channel are currently unknown.}
\label{fig:schematic}
\end{figure}

\subsection{Proton pathways}

% what are the channels
% where is the k-channel, what does it look like
% where is the d-channel, what does it look like
% the H-channel
% exit channel unknown
Two proton uptake pathways known as the K- and D-channels (see Figure \ref{fig:schematic}) have been discovered in bacterial and eukaryotic homologues of C\emph{c}O using site-directed mutagenesis, biophysical, and X-ray crystallographic techniques \cite{Tsukihara:1996p10224,Brzezinski:1998p5542,Pfitzner:1998p10266,SvenssonEk:2002p5595,Lee:2010p8614}. The K-channel, contained within subunit II, spans from E101 (\emph{R. sphaeroides} numbering) at the cytoplasm-exposed surface to Y288 at near the BNC, and contains a highly conserved residue K362, after which the channel is named \cite{Tomson:2003p10255}. This channel, as observed in several crystallographic structures C\emph{c}O, does not have a well-ordered string of water molecules as only two water molecules are typically seen \cite{Tsukihara:1996p10224,Qin:2008p7672,SvenssonEk:2002p5595}. However, mutations of conserved, polar residues within the K-channel have been shown to block or inhibit steady state oxidase activity, suggesting that it is a hydrated proton channel \cite{Pfitzner:1998p10266,Zaslavsky:1998p10317,Ganesan:2010p8417}. The D-channel is located in subunit I and spans 25 Å from residue D132 to residue E286. Residue D132 (which gives the channel its name) is at the surface of the enzyme exposed to the cytoplasm, and residue E286 is in the core of the enzyme, approximately 10 Å away from the BNC. The D-channel is believed to be hydrated by approximately 10 water molecules, as supported by both crystallographic structures \cite{SvenssonEk:2002p5595} and systematic computer simulations \cite{Henry:2009p4543}. The water molecules form a discontinuous water wire from D132 to E286, stabilized by hydrogen bonds with the side chains of residues N139, N121, N207, S142, S200, S201, and S197. This water wire is interrupted by the side chain of the highly conserved residue N139 \cite{Qin:2006p7708}. Residue N139 is one of two asparagine residues that line the D-channel approximately one-quarter of the way in, in what appears to be the narrowest portion of the D-channel. The proton pathway beyond E286, which is about 10 Å away from the BNC, is not evident from crystallographic structures.
A third proton channel, known as the H-channel, has also been observed in mammalian oxidase crystal structures but is believed not to be functionally relevant in prokaryotic oxidases \cite{Tsukihara:1996p10224,Lee:2000p10296}. The reason for the existence of multiple proton channels is not yet entirely clear \cite{Gennis:2004p10239}. Each of these three proton pathways are thought to be proton uptake channels, and since C\emph{c}O is a true proton pump, there must be at least one exit channel. However, the location of an exit channel has yet to be pinpointed within the enzyme \cite{Gennis:2004p10239,Sugitani:2009p5715}. The exit channel is presumed to be located ``above'' the two hemes where a many water molecules have been resolved in multiple crystal structures \cite{SvenssonEk:2002p5595,Qin:2006p7708,Muramoto:2007p8383,Durr:2008p6162,Koepke:2009p10494,Liu:2011p9691}.

% what do these pathways do and how do we study them?
From the requirements of the overall reaction catalyzed by C\emph{c}O, the proton channels must deliver a total of 8 protons: half of them to the binuclear centre (chemical), and the other half to the proton loading site (vectorial). The channels provide the physical medium for proton translocation, but it is the role of the catalytic cycle to provide the energetic conditions necessary to drive the protons in a single direction through them. Therefore, the channels must facilitate the coupling of redox chemistry to proton translocation while simultaneously preventing proton leakage. A common experimental approach for studying the channels has been to find mutations that affect proton translocation and/or the enzyme's turnover rate \cite{Konstantinov:1997p5292,Zaslavsky:1998p10317,Mills:2000p4585,Ganesan:2010p8417}. Studies based on this approach have afforded insight into the functional relevance of each channel by distinguishing critical residues required at each step of the catalytic cycle \cite{Konstantinov:1997p5292}.

% Specify the mutations and their effect on which portion of the catalytic cycle (refer to \ref{fig:catalytic_cycle})
% describe how we were able to deduce the two-channel concept
% start with the major inhibiting mutations
% key inhibiting mutations of the K-channel
Most studies of the K-channel have focused on the K362M mutant in \emph{R. sphaeroides} (K354M in the \emph{P. denitrificans} oxidase), as well as mutations of E101 and T359 \cite{Zaslavsky:1998p10317,Pfitzner:1998p10266,Ruitenberg:2000p10226,Mills:2000p4585,Ganesan:2010p8417}. Common features of K-channel mutants are a very low steady-state redox activity and a greatly decreased rate of reduction of heme $a_3$ when the oxidized enzyme is reduced anaerobically\footnote{Measured via time-resolved anaerobic reduction using an electron donor such as ascorbate.} \cite{Ganesan:2010p8417}. This implies that the loss of steady-state redox activity resulting from mutations within the K-channel is due to impairment of the reductive portion of the catalytic cycle. To ensure that such mutations do not affect the oxidative portion of the catalytic cycle, the reductive portion can be bypassed experimentally using \ce{H2O2} as the electron acceptor (instead of \ce{O2}), as \ce{H2O2} is already reduced by two electrons relative to \ce{O2} \cite{Zaslavsky:1998p10317}. The peroxidase reaction occurs only during the oxidative portion of the catalytic cycle \cite{Zaslavsky:1998p10317}. K-channel mutants exhibit full peroxidase activity, supporting the claim that K-channel mutants only inhibit the O$\rightarrow$R transition, whereby the reduction of active site is accompanied by the uptake of two protons prior to the cleavage of \ce{O2} (see Figure \ref{fig:catalytic_cycle}) \cite{Zaslavsky:1998p10317}. These studies suggest that, in wildtype oxidase, the main role of the K-channel is to deliver two protons to the BNC during the O$\rightarrow$R transition of the catalytic cycle \cite{Ganesan:2010p8417}.

% key inhibiting mutations of the D-channel
By contrast, several mutations in the D-channel decouple proton pumping from oxidase activity (see Table \ref{tbl:dchannel_mutations}) \cite{Pfitzner:2000p5484,Gennis:2004p10239}. Residue D132, at the entrance of the D-channel, has been proposed to act as a ``proton antenna'' that recruits protons from bulk water into the D-channel \cite{Mills:2000p4585,Henry:2009p4543}. Mutations of D132, such as D132N and D132A (\emph{R. sphaeroides}), exhibit approximately 5\% of wildtype redox activity without any proton pumping \cite{Fetter:1996p5464}. On the other hand, mutations of residue N139 further up the D-channel result in a 2- to 3-fold increase in redox activity compared to wildtype oxidase \cite{Pfitzner:2000p5484,Pawate:2002p5614,Namslauer:2003p6853}. These mutagenesis data provide a strong case that the D-channel is the primary conduit for pumped protons and that it is also involved in the delivery of at least two chemical protons (per turnover) following the binding of dioxygen to the BNC \cite{Brzezinski:1998p5542,Gennis:1998p5649,Ganesan:2010p8417}.

% DCHANNEL MUTATIONS TABLE START
\begin{table}
    \begin{center}
    \begin{singlespaced}
    \caption{Summary of oxygen redox activity and proton pumping in the wildtype and in D-channel site-directed mutants of cytochrome \emph{c} oxidase. Decoupling mutants are shown in bold. Single mutants are shown above double and triple mutants. All mutations are shown using \emph{R. sphaeroides} numbering.}
    \vspace{10pt}
    \label{tbl:dchannel_mutations}
    \footnotesize{
    \begin{tabular}{l|c|c|c|c}
    Organism  & Mutation & Activity (\% of WT) & Pumping & Ref. \\
    \hline
    \emph{R. sphaeroides} & WT & 100 & Yes & - \\
    \emph{R. sphaeroides} & \textbf{Y33H} & 40 & No & \cite{Namslauer:2010p9850} \\
    \emph{R. sphaeroides} & N121A & 110 & Yes & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{N121D} & 70 & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{N121T} & 100 & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{D132A} & $<5$ & No & \cite{Mills:2000p4585,Fetter:1995p6852} \\
    \emph{R. sphaeroides} & \textbf{D132N} & $<5$ & No & \cite{Mills:2000p4585} \\
    % \emph{E. coli} & D132N & 45 & Reduced & \cite{GarciaHorsman:1995p5547} \\ % From cytochrome bo3
    \emph{R. sphaeroides} & \textbf{N139A} & 40 & No & \cite{Zhu:2010p8237} \\
    \emph{P. denitrificans} & \textbf{N139A} & 11 & No & \cite{Durr:2008p6162} \\
    \emph{R. sphaeroides} & \textbf{N139C} & 110 & No & \cite{Zhu:2010p8237} \\
    \emph{P. denitrificans} & \textbf{N139C} & 86 & No & \cite{Durr:2008p6162} \\
    \emph{R. sphaeroides} & \textbf{N139D} & 150-300 & No & \cite{Pawate:2002p5614,Namslauer:2003p6853} \\
    \emph{P. denitrificans} & \textbf{N139D} & 100-115 & No & \cite{Pfitzner:2000p5484} \\
    % \emph{E. coli} & \underline{N139D} & 48 & Reduced & \cite{GarciaHorsman:1995p5547} \\ % From cytochrome bo3
    \emph{R. sphaeroides} & \textbf{N139E} & 110 & No & \cite{Zhu:2010p8237} \\
    \emph{P. denitrificans} & \textbf{N139E} & 78 & No & \cite{Durr:2008p6162} \\
    \emph{R. sphaeroides} & \textbf{N139L} & 7 & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & N139Q & 60 & Yes & \cite{Zhu:2010p8237} \\
    \emph{P. denitrificans} & N139Q & 52 & Yes & \cite{Durr:2008p6162} \\
    % \emph{E. coli} & N139Q & 109 & Yes & \cite{GarciaHorsman:1995p5547} \\
    \emph{R. sphaeroides} & \textbf{N139S} & 90 & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{N139T} & 40 & No & \cite{Lepp:2008p5615} \\
    \emph{P. denitrificans} & \textbf{N139V} & $<10$ & No & \cite{Pfitzner:2000p5484} \\
    % \emph{E. coli} & \underline{N139V} & 22 & Reduced & \cite{GarciaHorsman:1995p5547} \\ % From cytochrome bo3
    \emph{R. sphaeroides} & S142A & 80 & Yes & \cite{Zhu:2010p8237} \\
    \emph{P. denitrificans} & S142A & 96 & Yes & \cite{Pfitzner:2000p5484} \\
    \emph{R. sphaeroides} & \textbf{S142D} & $<5$ & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & S200I & 83 & Yes & \cite{Lee:2010p8614} \\
    \emph{R. sphaeroides} & \textbf{G204D} & $<5$ & No & \cite{Han:2005p5283} \\
    \emph{R. sphaeroides} & N207A & 90 & Yes & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{N207D} & 100 & No & \cite{Han:2006p5624} \\
    \emph{P. denitrificans} & \textbf{N207D} & 50 & No & \cite{Pfitzner:2000p5484} \\
    \emph{R. sphaeroides} & N207T & 90 & Yes & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & E286Q & $<1$ & No & \cite{Adelroth:1997p5385} \\
    \emph{P. denitrificans} & \textbf{N121V/N139D} & 75 & No & \cite{Pfitzner:2000p5484} \\
    \emph{R. sphaeroides} & D132N/N139D & 20 & Yes & \cite{Branden:2006p9874} \\
    % \emph{E. coli} & D132N/N139D & 33 & Yes & \cite{GarciaHorsman:1995p5547} \\ % From cytochrome bo3
    \emph{R. sphaeroides} & \textbf{D132N/N139T} & 90 & No & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & \textbf{D132N/S200I} & $<5$ & No & \cite{Lee:2010p8614} \\
    \emph{R. sphaeroides} & \textbf{D132N/S200V/S201V} & 7 & No & \cite{Lee:2010p8614} \\
    \emph{R. sphaeroides} & \textbf{D132N/S200V/S201Y} & 7 & No & \cite{Lee:2010p8614} \\
    \emph{R. sphaeroides} & S142A/N207A & 80 & Yes & \cite{Zhu:2010p8237} \\
    \emph{R. sphaeroides} & S200V/S201V & 37 & Yes & \cite{Lee:2010p8614} \\
    \emph{R. sphaeroides} & \textbf{S200V/S201Y} & 11 & No & \cite{Lee:2010p8614} \\
    \hline
    \end{tabular}
    }
    \end{singlespaced}
    \end{center}
\end{table}
% DCHANNEL MUTATIONS TABLE END

% now, bring focus on the D-channel's unusual feature
% what is the branch? how does it interact with the PLS?
The unusual feature of the D-channel, that it delivers both chemical and vectorial protons, requires that it utilize some sort of proton branching mechanism. This mechanism must be able to send protons either to the BNC (for redox chemistry), or to the PLS (for pumping), depending on the catalytic state of the enzyme. Spectroscopic and mutagenesis studies indicate that the side chain of E286 is implicated in this shuttling mechanism \cite{Pomes:1998p5611,Namslauer:2003p6853,Gennis:2004p10239}. For example, the E286Q mutant inhibits the enzyme's pumping and redox activity by removing the acid group at the top of the D-channel \cite{Adelroth:1997p5385}. This suggests that the protonation/deprotonation of E286 is critical to both pumping and redox activity. E286 is believed to be the branching point between proton pathways to the BNC and to the PLS \cite{Riistama:1997p8271,Adelroth:2000p5278} and it has even been suggested to play a role in the directionality of proton translocation through the enzyme as a valve that prevents proton leakage \cite{Siegbahn:2007p5285}. The catalytic cycle requires some sort of finely tuned timing for proton delivery. A delay in proton uptake from the P-side could potentially result in a failure of E286 to send vectorial protons off in the right direction. Therefore, the rate of proton uptake must be faster than the rate-limiting step of proton delivery to the redox site \cite{Zhu:2010p8237}. This is how proton pumping may be inhibited without slowing down the rate of the oxidase reaction, and this has been proposed to form the basis of the mechanism by which some decoupling mutants function \cite{Zhu:2010p8237}. Unfortunately, the molecular mechanism and the factors that control ``kinetic gating'' within the D-channel are not known. By studying the molecular mechanism of decoupling mutants, we can gain insight into the finely tuned branching mechanism and elucidate the molecular mechanism that couples redox chemistry and proton pumping.

\subsection{Putative decoupling mechanisms}

% Explain that decoupling mutants are different
% how do mutations cause decoupling?
Several proposals have been made to explain the molecular mechanism by which some D-channel mutations result in the decoupling of proton pumping from oxidase activity. However, none can consistently explain all of the experimental data \cite{Zhu:2010p8237}. All D-channel protons rely on residue E286 to shuttle them to either the BNC or PLS, so depending on the mutation, the decoupling mechanism may in principle be based on electrostatic or structural perturbations of E286. Electrostatic perturbations (especially in the N139D and N207D mutants) have been shown to increase the apparent p$K_a$ of E286, with the consequence of preventing the transfer of protons to the PLS but not to the BNC \cite{Namslauer:2003p6853,Han:2006p5624,Branden:2006p9874,Fadda:2008p5482,Lee:2010p8614}. Such perturbations may affect the metastable conformational states of residue E286 as well as the hydrogen bonded network of water molecules in close proximity to the residue, as observed in a few studies \cite{Vakkasoglu:2006p5432,Lepp:2008p5615,Durr:2008p6162,Zhu:2010p8237}. Local perturbations within the D-channel may slow down proton transfer below a threshold rate required for proton transfer to the PLS, but not to the BNC \cite{Mills:2000p4585,Henry:2009p4543,Zhu:2010p8237}. For example, the decoupled D132N mutation exhibits very low redox activity, likely due to a decrease in proton affinity resulting from the loss of the carboxylate group (whose charge may act as a ``proton antenna'') at the channel's entrance. In 2006, Brändén et al. \cite{Branden:2006p9874} showed that the combination of two decoupling mutations, N139D and D132N, resulted in a functional (``re-coupled'') double mutant. This finding provided insight into the mechanism of decoupling, attributed to shifts in the apparent p$K_a$ of residue E286. Residue E286 in wildtype C\emph{c}O (from \emph{R. sphaeroides}) has an apparent p$K_a$ of 9.4 \cite{Namslauer:2003p10505}, which is relatively high but not unusual for a carboxylic acid group in a hydrophobic environment. Several studies have shown, both through theory and experiment, that the p$K_a$ of E286 shifts upwards of 11 in the N139D mutant oxidase \cite{Branden:2006p9874,Fadda:2008p5482}. In the double mutant (N139D/D132N), however, the p$K_a$ of E286 was found to decrease to approximately 9.7, suggesting that it is the restoration of the wildtype p$K_a$ of E286 that leads to the ``re-coupled'' phenotype \cite{Branden:2006p9874}. Taken together, these results suggest that the effect of the N139D and D132N mutations are electrostatic in nature.

Consistent with a dominant role of electrostatic forces in the proper function of the D-channel, and indeed of the enzyme as a whole, there is also experimental evidence that divalent cations inhibit proton pumping in C\emph{c}O. Numerous studies have shown that \ce{Zn^{2+}} inhibits proton pumping and redox activity in C\emph{c}O by blocking the either D-channel or K-channel on the N-side of the membrane, or by blocking some exit pathway on the P-side \cite{Kannt:2001p8408,Aagaard:2002p8410,Kuznetsova:2005p8380,Faxen:2006p8398,Francia:2007p8388,Qin:2007p8387,Muramoto:2007p8383,Vygodina:2008p8389}. Multiple crystal structures have directly confirmed the binding of the inhibitory cations at or near the entrances of the D- and K-channels \cite{Qin:2007p8387,Muramoto:2007p8383}. In wildtype \emph{P. denitrificans} oxidase, \ce{Zn^{2+}} has been shown to decouple proton pumping from redox activity \cite{Kannt:2001p8408}. Interestingly, it has also been shown that the D132N mutant (\emph{R. sphaeroides}) is not inhibited by \ce{Zn^{2+}} \cite{Aagaard:2002p8410}. Whether cationic inhibition plays a role in the decoupling mechanism in D-channel mutants remains to be seen.

\section{Computer simulations of cytochrome \emph{c} oxidase}

% what has been studied using computer methods?
Elucidating the molecular mechanism of proton pumping in enzymes requires information on molecular motions that are unavailable from static crystallographic structures, but can be predicted from computer simulations. As a result, computational techniques have played a significant role in the study of cytochrome \emph{c} oxidase. Simulations of C\emph{c}O are particularly challenging as the enzyme contains a variety of interacting cofactors and intermediate ligands, and it is a large membrane protein complex in which small conformational changes may have significant effects on the hydration of proton channels and cavities. Nevertheless, many aspects of proton pumping have been examined through the study of protein dynamics on the atomistic scale. Quantum mechanical modelling techniques have been invaluable in the development of the model for catalytic activity and the interplay between the key residues in the active site \cite{Fadda:2008p5482,Johansson:2008p5653,Xie:2008p8091,Lee:2009p8180,Kamiya:2011p10246}, as well as in studies of protonation states and p$K_a$s of titratable residues within the enzyme \cite{Quenneville:2006p5306,Fadda:2008p5482,Leontyev:2009p9842}. Empirical molecular mechanics force fields, which approximate quantum mechanics, can be used to generate time trajectories using molecular dynamics (MD). MD-based approaches have been used in the study of the thermodynamic basis of hydration within proton channels and cavities \cite{Olkhova:2004p6851,Tashiro:2005p7174,Sugitani:2009p5715,Henry:2009p4543}, the functions of key residues such as E286, D132, and N139 \cite{Ghosh:2009p10110,Kaila:2009p6199,Henry:2009p4543}, as well as the movement of \ce{H^+} in the top half of the D-channel \cite{Xu:2005p9841,Xu:2008p9867}.

As a result of the inherent complexity and size of C\emph{c}O, most simulation-based studies have used finite-sized, conformationally biased models \cite{Tashiro:2005p7174,Xu:2008p9867,Ghosh:2009p10110,Sugitani:2009p5715,Henry:2009p4543,Kaila:2009p6199}. To date, only a single published study used an unrestrained model of C\emph{c}O embedded in a solvated lipid bilayer, though the total sampling time less than 1.5 ns \cite{Olkhova:2004p6851}. By contrast, over 2 microseconds of cumulative sampling was performed using a finite-sized system in a recent study done in our laboratory \cite{Henry:2011p10221}. In this study, the use of a simplified model resulted in high computational efficiency, allowing for more sampling. The finite-sized simulation model consisted of the protein (C\emph{c}O from \emph{R. sphaeroides}) \emph{in vacuo}, with a ``water cap'' mimicking bulk water at the entrance of the D-channel (see Figure \ref{fig:dchannel_full_zoom}). In finite-sized systems such as this, significant conformational restraints are required to stabilize the system in space (there are no lipids or water surrounding the protein) which may introduce biases that lead to erroneous results. Advancement in computer technology over the past decade has given us the opportunity to extend simulation timescales as well as increase the sizes of our models. As a result, large, unbiased simulations of both soluble and membrane-bound proteins are now becoming possible. In this thesis, a fully unbiased C\emph{c}O model is constructed to extend the scope of previous studies.

\begin{figure}[htbp]
\centering
\includegraphics{figures/dchannel_zoom/full_and_dchannel.png}
\caption[The finite-size simulated model of cytochrome \emph{c} oxidase from R. \emph{sphaeroides} used in two previous studies from our laboratory.]{The finite-size simulated model of cytochrome \emph{c} oxidase from R. \emph{sphaeroides} used in two previous studies from our laboratory \cite{Henry:2009p4543,Henry:2011p10221}. Subunit I, which contains the D channel (inset) and the binuclear centre (BNC), is highlighted in blue. Highlighted are heme $a$, heme $a_3$, Cu$_A$, Cu$_B$ and key residues of the D-channel, which extends from D132 to E286. This conformation is taken from one of our simulations with N139 in the open state and a chain of 12 water molecules in the D-channel. Also shown is the cap of 57 water molecules at the entrance of the D-channel used to represent bulk solution.}
\label{fig:dchannel_full_zoom}
\end{figure}

\section{Recent progress in elucidating the molecular mechanism of decoupling}

The decoupled phenotype has been proposed to arise from either electrostatic or structural perturbations within the D-channel. This fails to explain precisely what's happening at the molecular level. One important feature of the D-channel that has been glossed over by many studies is the gap in the water wire created by residue N139. A recent theoretical study done in our laboratory, based on simulations of the system shown in Figure \ref{fig:dchannel_full_zoom}, examined the relationship between the orientation of the side chain of residue N139 (defined by its $\chi_1$ dihedral angle) and functional hydration of the D-channel. The results predicted that residue N139 has at least two metastable orientations \cite{Henry:2009p4543}. In one orientation (called the ``open'' state), the water wire gap may be occupied by a water molecule. In the other orientation (the ``closed'' state), the gap is occupied by the side chain of N139 and is thus dehydrated. This finding suggests that the residue may be involved in a gating mechanism that controls the rate of proton uptake by repeatedly opening and closing the gap in the water wire \cite{Henry:2009p4543}. It follows that such a gate, which is conveniently located at the narrowest part of the D-channel, could also act as a ``proton filter'', ensuring the selectivity of the D-channel to proton uptake. If residue N139 does in fact play such a critical role, then mutations of the residue could certainly compromise the function of the D-channel.

In a follow-up study that we published this year \cite{Henry:2011p10221}, the favourable rotameric states of residue 139 in four variants (N139, N139D, D132N, and N139D/D132N) were determined using the same computational method. A third, ``outfacing'', state was found in all variants. The results showed that the ``closed'' conformer of residue 139 (as seen in the crystal structure) is the lowest energy state in the wildtype, N139D, and D132N systems. The ``outfacing'' state was found to be most favourable in the double mutant (N139D/D132N). It was also shown that the p$K_a$ of residue E286 is sensitive, not only to the charge density within the D-channel, but also to the orientation of residue 139 (see Figures \ref{fig:pmf_chi1_biased} and \ref{fig:pkas}, and Table \ref{tbl:e286_pkas}). The introduction of a negatively charged residue into the D-channel in the N139D variant increases the average p$K_a$ of E286, potentially affecting the proton branching mechanism. On the other hand, the p$K_a$ of E286 in the double mutant (N139D/D132N) in the ``outfacing'' state is equal to that in the wildtype N139 ``closed'' state \cite{Henry:2011p10221}. Also, the ``outfacing'' state of residue N139D in the double mutant (which lacks residue D132) may help restore proton uptake by acting as a ``proton antenna'', similar to residue D132 in wildtype oxidase. These results suggest a mechanism for the restorative effect in the double mutant, as the magnitude of p$K_a$ shifting depends on the balance of charges in the D-channel as well as the side chain orientation of residue 139.

\begin{figure}[htbp]
\centering
\includegraphics{figures/chi1_pmfs_biased/pmf.png}
\caption[PMF for the $\chi_1$ rotation of the side chain of residue 139 in wildtype and mutant cytochrome \emph{c} oxidase in a simplified simulation model.]{PMF for the $\chi_1$ rotation of the side chain of residue N139 (blue), N139D in the D132N/N139D double mutant (green), N139D (red), and N139 in the D132N mutant (cyan). The three metastable states are labeled above the graph. Data is from Henry et al. (2011) \cite{Henry:2011p10221}.}
\label{fig:pmf_chi1_biased}
\end{figure}

% pKa of E286 plots, data from Henry 2011
\begin{figure}[htbp]
\centering
\includegraphics{figures/pkas/pkas.png}
\caption[p$K_a$ of E286 in successive catalytic steps of various C\emph{c}O models, with residue 139 in different orientations.]{p$K_a$ of E286 in successive catalytic steps of various C\emph{c}O models, with residue 139 in different orientations: C for ``closed'', and O for ``outfacing''. The charge distribution of the two active site moieties is represented by the symbols below the x-axis: [0$\mid$1], [1$\mid$0] and [1$\mid$1]. The catalytic states corresponding to proton delivery from E286 are shown in bold. Data is from Henry et al. (2011) \cite{Henry:2011p10221}.}
\label{fig:pkas}
\end{figure}

\begin{table}
    \begin{center}
    \begin{singlespaced}
    \caption{Average p$K_a$ of E286 in proton-loading states in all systems with residue 139 in ``outfacing'' and ``closed'' orientations. ``Outfacing'' orientations for wildtype and D132N systems were not computed. Data is from Henry et al. (2011) \cite{Henry:2011p10221}.}
    \label{tbl:e286_pkas}
    \vspace{10pt}
    \begin{tabular}{lccc}
            & \multicolumn{2}{c}{p$K_a$ of E286} \\
    System  & 139 ``outfacing'' & 139 ``closed'' \\
    \hline
    Wildtype & - & 10.8 \\
    D132N & - & 9.9 \\
    N139D & 11.2 & 11.7 \\
    N139D/D132N & 10.8 & 11.1 \\
    \hline
    \end{tabular}
    \end{singlespaced}
    \end{center}
\end{table}

This study provides a structural and electrostatics-based explanation for decoupling and re-coupling of proton pumping and redox chemistry in cytochrome \emph{c} oxidase mutants. However, while it suggests how gating and electrostatics may play a key role in the decoupling mechanism, it falls short of explaining how redox activity or ``turnover'' is modulated in these mutants (see Table \ref{tbl:dchannel_mutations}).

\section{Objectives}

Despite C\emph{c}O's significance as a core respiratory protein and the tremendous amount of existing research, it remains poorly understood. Specifically, the details of redox-coupled proton pumping are still controversial. Understanding the molecular mechanism hinges on acquiring more detailed structural information about catalytic intermediates, amino acid side chain conformations and properties of water molecules forming proton-relay pathways in the enzyme's interior. Major gaps in our knowledge of C\emph{c}O include (but are not limited to):

\begin{itemize}
\begin{singlespaced}    
  \item locating the proton loading site,
  \item the proton branching mechanism of residue E286,
  \item hydration states of channels and cavities throughout the catalytic cycle,
  \item proton selectivity within the D-channel,
  \item the proton uptake mechanism of the K-channel, 
  \item and the molecular mechanisms of coupling and decoupling.
\end{singlespaced}
\end{itemize}

% How some of these problems will be solved
In this thesis, computer simulations are used to investigate proton gating and selectivity in wildtype and mutant cytochrome \emph{c} oxidase from \emph{R. sphaeroides}. The previous simulation system was finite-sized, whereby the whole protein was restrained \emph{in vacuo} except for the region of interest (the D-channel). Such restraints are necessary to prevent the structure from falling apart during the simulation, as it is not embedded in a lipid bilayer, nor solvated in water. These conformational restraints may have affected energetic barriers in the region of interest and may have improperly sampled certain states. To expand on previous work, a new unbiased model of C\emph{c}O embedded in a solvated lipid bilayer is constructed and used in three studies. As in previous studies, the focus is on residue N139 and single and double mutants such as N139D, D132N, N139D/D132N, and N139A. Over 5 $\mu s$ of cumulative sampling is performed using these new systems across three studies. Conformational analysis of residue 139 is performed in the new system as a comparison with earlier results from the simplified system. Hydration of the D-channel and K-channel is analyzed from the time trajectories and a fully hydrated K-channel is seen for the first time in a molecular dynamics simulation. In addition, the new model unexpectedly reveals a previously unknown mechanism by which the N139D mutant may impair proton selectivity. During the simulation of the unrestrained N139D system, I observed binding of a \ce{K^+} ion to the entrance of the D-channel. This finding led to an investigation of \ce{K^+} binding to the D-channel in C\emph{c}O variants. As mentioned above, it has been proposed that the highly-conserved residue N139 acts as a barrier to cations, imparting proton selectivity to the D-channel \cite{Henry:2009p4543}. The results of this study show that the N139D mutation may abolish the ability for the D-channel to block other cations, potentially leading to inhibition of pumping by cations such as \ce{K^+}.
